collect_performance.py

```diff
@@ -121,19 +208,18 @@ def get_global_performance(eval_file: str|os.PathLike, save_to: str|os.PathLike
         df.to_html(save_to / f'{file_stem}.html')
         df.to_excel(save_to / f'{file_stem}.xlsx')
 
-        
-
 if __name__ == "__main__":
```

Comment:
it'll be nice to move this to a main() function for reusability of the script and its functions.